### ISSUE TITLE
How to divide the synthetic calcium imaging data into training set and test set？

### ISSUE THREAD

**Issue by HCY124** [cabooster/DeepCAD-RT#14]
Hi cabooster! I noticed that the synthetic calcium imaging dataset contains 14 images. Could you please tell me which images are included in the training set and test set? Thank you very much.

**Comment by cabooster** [comment]
Hi! Our method is a self-supervised denoising algorithm, which means the training set is the dataset you want to process. So you can choose arbitrary noisy image as trainng/test set. For more details, please see the companion paper where the method first appeared: ["Real-time denoising of fluorescence time-lapse imaging enables high-sensitivity observations of biological dynamics beyond the shot-noise limit, bioRxiv (2022)"](https://www.biorxiv.org/content/10.1101/2022.03.14.484230v1).

**Comment by HCY124** [comment]
Thank you for your reply！
